src/components/dropdownSearch/DropdownSearchReducers.tsx

```diff
@@ -111,11 +154,17 @@ export const dropdownSearchReducer = (state: IDropdownSearchState = dropdownSear
         return {
           ...state,
           id: action.payload.id,
-          selectedOption: state.activeOption,
           isOpened: false,
+          selectedOptions: state.selectedOptions.push(state.activeOption),
```

Comment:
```
selectedOptions: [
  ...state.selectedOptions,
  _.extend({}, state.activeOption),
],
```
State is read-only and immutable (http://redux.js.org/docs/recipes/reducers/ImmutableUpdatePatterns.html)